galleries/tutorials/artists.py

```diff
@@ -11,15 +11,15 @@
 
 There are three layers to the Matplotlib API.
 
-* the :class:`matplotlib.backend_bases.FigureCanvas` is the area onto which
+* the :class:`!matplotlib.backend_bases.FigureCanvas` is the area onto which
   the figure is drawn
-* the :class:`matplotlib.backend_bases.Renderer` is the object which knows how
-  to draw on the :class:`~matplotlib.backend_bases.FigureCanvas`
+* the :class:`!matplotlib.backend_bases.Renderer` is the object which knows how
```

Comment:
Same issue as above: *Renderer* is a concept realized through the subclasses of `RendererBase`.